net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A VPN command-line tool from protonvpn - python rewrite"
+HOMEPAGE="(
+https://protonvpn.com
+https://github.com/ProtonVPN/protonvpn-cli-ng
+)"
+SRC_URI="https://github.com/ProtonVPN/${PN}-ng/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+KEYWORDS="~amd64"
+SLOT=0
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/setuptools
+	dev-python/docopt
+	dev-python/requests
+	dev-python/pythondialog:0
```

Comment:
You can drop the `PYTHON_DEPS`, but you **need** to add `PYTHON_USEDEP` to these python deps. I checked, and some of the deps are only available for python3.6. So you need to drop support for python3.7 and python3.8 until the deps are python-enabled correctly. 

You can check these with repoman prior to committing.

Reply:
Ahh, now it's making more sense.  Thank you for checking into that.

Reply:
Ahh, now it's making more sense.  Thank you for checking into that.
However, this also means that I probably have to use python_single_target.